src/toolbar.scss

```diff
@@ -142,66 +158,60 @@ $title-toolbar-height: 2.75rem;
   }
 
   &__overflow {
-    &.#{$fd-namespace}-popover {
-      .#{$block}__overflow__body {
-        @include fd-reset();
-
-        display: flex;
-        flex-direction: column;
-        max-width: 20rem;
-        padding: 0.1875rem 0.375rem;
-
-        > * {
-          display: block;
-          text-align: left;
-          width: auto;
-          margin: 0.1875rem 0;
-
-          @include fd-rtl() {
-            text-align: right;
-          }
+    &--body {
+      @include fd-reset();
+
+      display: flex;
+      flex-direction: column;
+      max-width: 20rem;
+      padding: 0.1875rem 0.375rem;
+      max-height: 50vh;
+      overflow: auto;
+
+      > .#{$block}__button {
```

Comment:
please avoid this. Is `>` really necessary?

Reply:
Modified.